tests/unit/test_pynblint.py

```diff
@@ -1,6 +1,154 @@
 from pynblint.notebook import Notebook
 from pathlib import Path
+import pytest
 
 if __name__ == '__main__':
-    nb = Notebook(Path('../fixtures/my-attempt-at-analytics-vidhya-job-a-thon.ipynb'))
-    print(nb.get_pynblint_results())
+    pytest.main()
+
+
+@pytest.fixture(scope="module")
+def notebooks():
+    nb1 = Notebook(Path("../fixtures") / Path("EmptyNotebook.ipynb"))
+    nb2 = Notebook(Path("../fixtures") / Path("FullNotebook.ipynb"))
+    nb3 = Notebook(Path("../fixtures") / Path("JustMarkdown.ipynb"))
+    nb4 = Notebook(Path("../fixtures") / Path("DefectedNotebook.ipynb"))
+    return {
+        "EmptyNotebook.ipynb": nb1,
```

Comment:
Why do you call this "EmptyNotebook" if it is not empty? It is quite misleading..